examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/WebDemoWithEventHub.csproj

```diff
@@ -3,8 +3,13 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="3.0.*" />
     <PackageReference Include="Azure.Identity" Version="1.1.1" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.1" />
+    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.3" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.3" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.3" />
```

Comment:
As we discussed, we shouldn't need to include the 3 packages above. They will be brought in automatically by dependencies.

Reply:
Removed these.

---
In reply to: [426103640](https://github.com/Azure/AppConfiguration/pull/329#discussion_r426103640) [](ancestors = 426103640)